### ISSUE TITLE
falling back to default configuration if mss_settings.json is missing

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#89]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/89) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

If rhere is no mss_settings.json stored, we want to use default values of default configurarion.

An additional  mss_settings.json  should not become mandatory

```
#!python


Traceback (most recent call last):
  File "/home/user/projects/mss_standard_name/mslib/msui/mss_pyui.py", line 562, in <module>
    main()
  File "/home/user/projects/mss_standard_name/mslib/msui/mss_pyui.py", line 544, in main
    mainwindow.createNewFlightTrack(activate=True)
  File "/home/iuser/projects/mss_standard_name/mslib/msui/mss_pyui.py", line 383, in createNewFlightTrack
    waypoints = config_loader(dataset="new_flighttrack_template", default=mss_default.new_flighttrack_template)
  File "/home/iuser/projects/mss_standard_name/mslib/mss_util.py", line 62, in config_loader
    with open(os.path.join(config_file)) as source:
TypeError: coercing to Unicode: need string or buffer, NoneType found
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

falling back to default configuration if mss_settings.json is missing, fixes #89

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #123)

falling back to default configuration if mss_settings.json is missing, fixes #89
